phpstan.neon

```diff
@@ -48,3 +51,9 @@ parameters:
         # needed for instanceof method API
         - '#Doing instanceof PHPStan\\Type\\ObjectType is error\-prone and deprecated\. Use Type\:\:isObject\(\) or Type\:\:getObjectClassNames\(\) instead#'
         - '#Doing instanceof PHPStan\\Type\\(Generic\\GenericObjectType|ArrayType) is error\-prone and deprecated#'
+
+        -
+            identifier: return.unusedType
+
```

Comment:
it shows returns constant definition

```
  :167   Method Rector\Doctrine\CodeQuality\Rector\Property\TypedPropertyFromToOneRelationTypeRector::provideMinPhpVersion() never returns 100000 so it can be removed from the  
         return type.                                                                                                                                                            
         🪪  return.unusedType  

 :50    Method Rector\Doctrine\CodeQuality\AttributeTransformer\ClassAttributeTransformer\InheritanceClassAttributeTransformer::getClassName() never returns 'Gedmo\\Mapping…'  
         so it can be removed from the return type.  
```

which seems valid as I see, I just updated to register the exact message instead https://github.com/rectorphp/rector-doctrine/pull/349/commits/2fd2a2ba6da7a66de518023837ee9ea56a364286